app/components/project-form.tsx

```diff
@@ -11,9 +11,10 @@ import { usePopoverContext } from "./popover";
 interface Props {
 	onClose: () => void;
 	project?: Project;
+	editMode?: boolean;
 }
 
-export function ProjectForm({ onClose, project }: Props) {
+export function ProjectForm({ onClose, project, editMode }: Props) {
```

Comment:
You don't need `editMode` to tell you if you're in edit mode. If the project is _truthy_ that's edit mode